### PR TITLE
Expose user and space membership information

### PR DESCRIPTION
**Pull request by geigerzaehler**
See the updated api docs for more information

### CODE REVIEW

**Comment by NickLiffen** [comment]
Hey, are we waiting for a code review here? 🤔 What needs to be done to get this merged?

**Comment by geigerzaehler** [comment]
We are waiting for feedback from users. Expect this to be merged next week.

**Comment by martzoukos** [comment]
Hey @NickLiffen , just to clarify what Thomas meant by "users", it's the developers of your team that we'd like them to have a look at the codebase to make sure that the [information we expose](https://github.com/contentful/ui-extensions-sdk/pull/151/files#diff-0b4574f1723ec32a5062748566bd603fR276) can help them solve their problems. After that 👍  we can then merge it.

Thanks! :)

**Comment by ricka** [comment]
Looks good from our end. :shipit:

File: docs/ui-extensions-sdk-frontend.md [hunk 1]
```diff
@@ -366,6 +394,7 @@ not available.
 
 _Since v3.1.0_
 
+[cma-docs]: https://www.contentful.com/developers/docs/references/content-management-api
```
Comment (jelz) [inline docs/ui-extensions-sdk-frontend.md]: Add a trialing slash so they don't have to follow redirect.

**Comment by geigerzaehler** [comment]
@ricka: We had some discussions and decided to remove the `policies` and `permissions` objects from the roles since they are low-level. Does this still work for you?

**Comment by ricka** [comment]
@geigerzaehler We believe this should be ok for our uses.  Thanks for the heads up.

**Pull request merged**